README.md

```diff
@@ -1,157 +1,179 @@
-# SignalFx-Tracing Library for Ruby: An OpenTracing Auto-Instrumentor
+# SignalFx Tracing Library for Ruby
 
-This utility provides users with the ability of automatically configuring OpenTracing community-contributed [instrumentation libraries](https://github.com/opentracing-contrib) for their Ruby application via a single function.
+The SignalFx Tracing Library for Ruby helps you instrument Ruby applications
+with the OpenTracing API to capture and report distributed traces to SignalFx.
 
-```ruby
-require 'signalfx/tracing'
-
-SignalFx::Tracing::Instrumenter.configure(auto_instrument:true)
-```
-
-## Installation
+## Requirements and supported software
 
-### General installation
+Instrumentation for routes using the following web servers is provided through
```

Comment:
I'd suggest not making this the 2nd paragraph since it's fairly context dependent (Supported servers list you moved below) and not an introduction to this library.  It also needs to be clarified since it's referring to the rack-tracer library and not rack itself.